Sources/MountebankSwift/Models/Stub/Response/Behavior.Copy.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+extension Behavior {
+    public struct Copy: Equatable, Codable {
+        public struct Using: Equatable, Codable {
+            public enum Method: String, Equatable, Codable {
+                case regex
+                case xpath
+                case jsonpath
+            }
```

Comment:
Did you consider putting the required properties in this enum to make the api more type safe and perhaps a bit more user friendly? Eg: 

```
public enum Using: String, Equatable, Codable {
    case regex(selector: String, options: Options)
    case xpath(selector: String, namespace: JSON, options: Options)
    case jsonpath(selector: String, options: Options)
}
```

Reply:
Perhaps you could remove the Using type alltogether and move everything directly into Behaviour

```
public enum Behavior: Equatable {
    case wait(miliseconds: Int)
    case waitJavascript(String)
    case copy(from: String, into: String, using: Method)
    ...
}

public enum BehaviorMethod: String, Equatable, Codable {
    case regex(selector: String, options: Options)
    case xpath(selector: String, namespace: JSON)
    case jsonpath(selector: String)
}
```

I think that would make great api: 

```
Behavior.copy(
    from: "path",
    into: "${code}",
    using: .regex(selector: "\\d+")
)
```

Instead of: 

```
Behavior.copy(
    Copy(
        from: "path",
        into: "${code}",
        using: Behavior.Copy.Using(method: .regex, selector: "\\d+")
    )
)
```